readme.md

```diff
@@ -24,3 +24,29 @@ from tcii18n import Translator
 translator = Translator('my_file.csv')
 translator.translate('My string')
 ```
+
+In a flask project:
+
+*core.py*
+
+```
+from tcii18n.template import flask_methods
+
+def get_translations_file():
+    return 'path/to/my/file.csv'
+
+flask_methods(app, get_translations_file)
+```
+
+*template.html*
+
+```
+<h1>{{ _("Sentence") }}</h1>
+```
+
+Where:
+
+-get_translations_file() is a function that returns the appropriate translation file path
```

Comment:
-get_translations_file() > add a space after "-" and use backticks _`_ for showing it as code.